repository/songs_list.json

```diff
@@ -0,0 +1,8 @@
+[
+    {"title":"Title 1", "artist":"Artist 1", "genre":"Genre 1"},
+    {"title":"Title 2", "artist":"Artist 2", "genre":"Genre 2"},
+    {"title":"Title 3", "artist":"Artist 3", "genre":"Genre 3"},
+    {"title":"Title 4", "artist":"Artist 4", "genre":"Genre 3"},
+    {"title":"Title 5", "artist":"Artist 5", "genre":"Genre 2"},
+    {"title":"Title 6", "artist":"Artist 6", "genre":"Genre 3"}
+]
```

Comment:
Please make sure to add non-blank line, terminated with a single LF (https://www.php-fig.org/psr/psr-12/#22-files).